mentoring/templates/html/ratingblock.html

```diff
@@ -6,38 +6,70 @@
   <div class="choices-list">
     <div class="choice">
       <div class="choice-result fa icon-2x"></div>
-      <label><input class="choice-selector" type="radio" name="{{ self.name }}" value="1"{% if self.student_choice == '1' %} checked{% endif %} />1</label>
-      <span class="low"> - {{ self.low }}</span>
-      <div class="choice-tips"></div>
+      <label class="choice-label">
```

Comment:
I didn't realize how similar these blocks of HTML were until you broke them up into shorter lines. Can you use a loop here, to avoid repeating yourself?
```html+django
<div class="choices-list">
{% for value in [1, 2, 3, 4, 5] %}
  <div class="choice">
    <div class="choice-result fa icon-2x"></div>
    <label class="choice-label">
      <input class="choice-selector" type="radio" name="{{ self.name }}" value="{{ value }}"
        {% if self.student_choice == value %} checked{% endif %} />
      {{ value }}
      {% if forloop.first %}
        - {{ self.low }}
      {% endif %}
      {% if forloop.last %}
        - {{ self.high }}
      {% endif %}
    </label>
    <div class="choice-tips-container">
      <div class="choice-tips"></div>
    </div>
  </div>
{% endfor %}
</div>
```
The `forloop.first` and `forloop.last` variables are set by Django's template system, to determine if you're on the first or last iteration of the loop. [Docs here.](https://docs.djangoproject.com/en/1.4/ref/templates/builtins/#for)